go-apps/meep-wais/api/swagger.yaml

```diff
@@ -1,2715 +1,2192 @@
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11STBCCTSSuccessCount:
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11TransmittedAMPDUCount:
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11TransmittedAMSDUCount:
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11TransmittedFragmentCount:
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11TransmittedFrameCount:
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11TransmittedFramesInGrantedRDGCount:
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11TransmittedMPDUsInAMPDUCount:
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11TransmittedOctetsInAMPDUCount:
+          format: int64
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint64
+        dot11TransmittedOctetsInAMSDUCount:
+          format: int64
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11TransmittedOctetsInGrantedRDGCount:
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11TwentyMHzFrameReceivedCount:
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11TwentyMHzFrameTransmittedCount:
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11nonSTBCCTSFailureCount:
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+        dot11nonSTBCCTSSuccessCount:
+          format: int32
+          type: integer
+          x-etsi-mec-cardinality: 0..1
+          x-etsi-mec-origin-type: Uint32
+      type: object
+    SubscriptionLinkList:
+      properties:
+        _links:
+          description: List of hyperlinks related to the resource.
+          properties:
+            self:
+              # description': ''
+              # x-etsi-mec-cardinality': '1'
+              # x-etsi-mec-origin-type': LinkType
+              $ref: '#/components/schemas/LinkType'
+            subscription:
```

Comment:
This is different from ETSI specification. Is this update necessary for implementation?